README.md

```diff
@@ -39,79 +39,96 @@ const client = new ApolloClient({
 ```
 
 ## Options
-```js
-const defaultOptions = {
+```typescript
+export interface ApolloLinkSentryOptions {
+  /**
+   * Determines if the given operation should be handled or discarded.
+   *
+   * If undefined, all operations will be included.
+   */
+  shouldHandleOperation: undefined | ((operation: Operation) => boolean);
```

Comment:
This definition style:
- shows the possible options
- indicates the default
- allows for a single point of truth that can also be navigated to from inside the IDE